packages/web/components/place-limit-tool/index.tsx

```diff
@@ -120,7 +140,12 @@ export const PlaceLimitTool: FunctionComponent<PlaceLimitToolProps> = observer(
     };
 
     const buttonText = useMemo(() => {
-      if (swapState.error) {
+      if (
+        swapState.error &&
+        swapState.error !== "errors.zeroAmount" &&
+        swapState.error !== "errors.emptyAmount" &&
+        swapState.error !== "limitOrders.insufficientFunds"
```

Comment:
can't you invert this whole chain of "!==" into one expression of "===" with the only leftover condition?

Reply:
Isn't exclusion simpler here? If we add more errors to the `swapState` we would have to update this every time right?

Reply:
I do not get if this boolean chains aims to isolate a specific error or not

Reply:
We don't want to display error text for certain errors right? Like Zero amount input/Empty input/Insufficient balance

Reply:
```typescript
const NON_DISPLAY_ERRORS = [
  "errors.zeroAmount",
  "errors.emptyAmount",
  "limitOrders.insufficientFunds",
];

const buttonText = useMemo(() => {
  if (swapState.error && !NON_DISPLAY_ERRORS.includes(swapState.error)) {
    return t(swapState.error);
  } else {
    return orderDirection === "bid"
      ? t("portfolio.buy")
      : t("limitOrders.sell");
  }
}, [orderDirection, swapState.error, t]);
```

How about something like this?